modules/public/views.py

```diff
@@ -162,28 +162,70 @@ def convert_limit_offset_to_size_page(link):
 
 
 def homepage():
+    featured_snaps = []
+    errors = []
+    status_code = 200
+    try:
+        featured_snaps = normalize_searched_snaps(api.get_featured_snaps())
+    except api.ApiResponseError as api_error_exception:
+        if hasattr(api_error_exception, 'errors'):
```

Comment:
Building on my [previous comment](https://github.com/canonical-websites/snapcraft.io/pull/376/files#r174540058), we shouldn't need to infer the type of an error that we created ourselves. If we're interested in the difference between an error list and another type of error (which we are) we should just make explicit error types to test against.